ApiUsage/ArmBased/README.md

```diff
@@ -1,25 +1,25 @@
 ---
 page_type: How to use AVAM API
 languages:
-- C#
-- .Net
+  - C#
+  - .Net
 products:
-- azure-video-analyzer-for-media
+  - azure-video-analyzer-for-media
```

Comment:
Should the name be changed to VI? (maybe this is a task to update all our samples)

Reply:
Yes we should!

Reply:
what is the official new name ? 
Azure Video Indexer ? 
Video Indexer  ?